pkg/apis/clusterregistry/register.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 // GroupName is the group name use in this package
-const GroupName = "clusterregistry"
+const GroupName = "clusterregistry.k8s.io"
```

Comment:
FYI, I also took care of this in my commit yesterday https://github.com/kubernetes/cluster-registry/pull/65/commits/49a3c800c87ad0b8e094781aa9d3052eb4364490 based on your feedback for #65.

Reply:
Oh, sweet! Will this cause you trouble if I commit this now?

Reply:
No that's fine. I can rebase.